src/Microsoft.DotNet.ProjectModel/Graph/LockFileReader.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Microsoft.DotNet.ProjectModel.Graph
 {
-    internal static class LockFileReader
+    public static class LockFileReader
```

Comment:
I'm not sure how I feel about this. In general I don't mind making this public, but I'm not sure I want publish to use it directly. Perhaps something more like how [`ProjectContext.CreateContextForEachFramework`](https://github.com/dotnet/cli/blob/master/src/Microsoft.DotNet.ProjectModel/ProjectContext.cs#L86) works to remove the direct processing of the lock file back into `ProjectContext`? `ProjectContext.CreateContextForEachTarget`?